### PR TITLE
Fix generative-aws module, add support for all Bedrock models

### PR DESCRIPTION
**Pull request by antas-marcin**
### What's being changed:

Fix generative-aws module, add support for all Bedrock models

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: modules/generative-aws/config/class_settings_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 )
 
 func Test_classSettings_Validate(t *testing.T) {
-	t.Skip("Skipping this test for now")
+	// t.Skip("Skipping this test for now")
```
Comment (parkerduckworth) [inline modules/generative-aws/config/class_settings_test.go:23]: Should this just be removed entirely?

File: modules/generative-aws/module.go [hunk 2]
```diff
@@ -67,8 +67,8 @@ func (m *GenerativeAWSModule) initAdditional(ctx context.Context, timeout time.D
 ) error {
 	awsAccessKey := m.getAWSAccessKey()
 	awsSecret := m.getAWSSecretAccessKey()
-
-	client := clients.New(awsAccessKey, awsSecret, timeout, logger)
+	awsSessionToken := os.Getenv("AWS_SESSION_TOKEN")
```
Comment (parkerduckworth) [inline modules/generative-aws/module.go:70]: Does this require docs change?

File: modules/generative-aws/config/class_settings_test.go (hunk 1 above)
Comment (antas-marcin) [reply modules/generative-aws/config/class_settings_test.go:23]: ahhh, yes, I have to remove it

File: modules/generative-aws/module.go (hunk 2 above)
Comment (antas-marcin) [reply modules/generative-aws/module.go:70]: good point, yes it needs

**Review by parkerduckworth — APPROVED** [review]

**Pull request merged**